pkgs/top-level/emacs-packages.nix

```diff
@@ -385,9 +385,9 @@ let
 in
   lib.makeScope newScope (self:
     {}
-    // melpaPackages self
     // elpaPackages self
     // melpaStablePackages self
```

Comment:
Every package in `melpaStablePackages` is also in `melpaPackages`, so we can remove this line to improve evaluation performance.

Reply:
The issue is some people might be using the ```nixpkgs.emacsPackagesNg.melpaStablePackages``` attribute set. This will at least preserve that for them until a better solution can be found.

Reply:
I'm not saying to remove `melpaStablePackages`, I'm saying to stop inheriting everything from it into the default package set.

Reply:
Okay did that, thanks! I'm not 100% on how this all works but I'm not sure if there is anyway to access "melpaStablePackages" now that it is not in the scope. At least this now fails:

```
nix-build -A emacsPackagesNg.melpaStablePackages.magit
```